### DISCUSSION TITLE
全球首款噴氣式飛行摩托車

### DISCUSSION THREAD

**Discussion by HSU-YU-JEN** [1101-GS4719-C/Python-Community-GS4719-C#88] [科技新知]
![image](https://user-images.githubusercontent.com/90545550/137093416-45a0fd43-5ade-4dbd-96db-e59f0949e148.png)

**噴氣背包航空公司Jetpack Aviation (JPA)** 最近研發一款**噴氣式飛行摩托車**，重約136 公斤，最高承載約 270 公斤，

且機車前方大螢幕內建雙向航空無線電系統，可查看導航、高度等資訊，且無須駕照即可飛行。

預計將在 2023 年上市，售價約38 萬美元(大約一千萬台幣)。

介紹影片:https://www.youtube.com/watch?v=URgznwTph6M&t=3s

**Comment by enoch-lightning** [comment]
看起來很酷，不過，我覺得不實用。第一是太貴，第二是都市這麼擠，可能飛一飛就撞到東西。第三是廢氣都是往天空排，飛在天空中，剛好當人體空氣清淨機。:smile:

**Comment by Jie2222** [comment]
交通方式從2D變成3D，所有現有的標準跟法規都要重新修改，安全跟隱私也是很大的考量

**Comment by swerfg03joe** [comment]
這種東西一看就知道是噱頭產物，並沒有實際的意義，說是要解決交通問題，但很明顯相關法規都沒有，且都市都有限高和禁航區的畫分，再者價格實在是太高，不符成本效益

**Comment by Twenty0522** [comment]
看影片好像在看電影演的未來世界超級酷的，但如果能體驗我還是會覺得很害怕，我的清單其中一項是飛行傘或跳傘，若是體驗過後或許會對噴氣式摩托車有不同的想法。

**Comment by harrylee1971** [comment]
花一千多萬去買一台摩托車不值得吧，建造出來一個不安全的交通工具對於普遍民眾不太合適

**Comment by 12Jesse24** [comment]
看起來真的超酷的，不過這東西一出來肯定會引發一系列問題，像是安全、治安、交通，甚至是隱私問題，所以也不太期待真的會有這東西被發明出來，而且感覺掉下來的話就死掉了，那這樣是要誰負責，是我嗎還是發明的人。

**Comment by Yanxxx7979** [comment]
感覺就算公司發布這項產品，大部分國家沒有修法的狀態根本無法實際上路

**Comment by Chengbosco** [comment]
感覺是智商稅，做得出來但很難用法律規範，到時候連天上都要出車禍了，有夠恐怖。

**Comment by THH1118** [comment]
噴氣式結果下面的路人都在吸這些氣體www,感覺可行性不太高

**Comment by mrweixd** [comment]
這項發明不論是否能在近期實現，多少都燃起了對未來的想像。不過現實層面要考慮的法規確實要先完善才比較有可能性。

**Comment by davidteng00000** [comment]
我很樂見這項充滿未來感的科技蓬勃發展，不過其實型方式還有待商榷，畢竟如果有危險駕駛，可能會造成很多危險，例如在假裡面睡覺的時候忽然有一台噴氣式飛行摩托車砸到你家屋頂，這其實蠻恐怖的

**Comment by chuanmin0317** [comment]
雖然這台摩托車的外型非常酷炫，但是相比傳統機車，這種新型交通工具的安全性有待考量，不過按照它的定價來說，也不是我這種人可以玩得起的，玩得起的人肯定能有非常好的安全措施避免危險吧

**Comment by TobywithDino** [comment]
看起來超級有趣，如果我以後發財致富，成為有錢人，一定也要買一台，喔! 不只買一台，還要買五台送給我的爸爸媽媽和三個白癡室友，讓室友在高空中體會生死的快感

**Comment by William08290829** [comment]
感覺很像GTA裡面的暴君，只差裝個追蹤飛彈就完成了。但這感覺在台灣開，颱風天直接飛到日本。這是個蠻有趣的設計但感覺做出來世界上也不知道哪裡需要這台ww，可能就真的圓滿大家的GTA夢吧。
